lib/spack/spack/relocate.py

```diff
@@ -320,6 +367,39 @@ def match_func(cpath):
     return
 
 
+def macholib_get_paths(cur_path):
+    """
+    Get rpaths, dependencies and id of mach-o objects
+    using py-macholib
+    """
+    try:
+        from macholib.MachO import MachO
+    except ImportError as e:
+        raise MissingMacholibException(e)
+    dll = MachO(cur_path)
+
+    import macholib.mach_o
```

Comment:
Can this import be done at top level?

Reply:
Yes. This code was put in before I vendored macholib.